2021/3xxx/CVE-2021-3708.json

```diff
@@ -4,15 +4,69 @@
     "data_version": "4.0",
```

Comment:
('Expecting property name enclosed in double quotes: line 51 column 13 (char 1689)',)